api/filter/network/tcp_proxy.proto

```diff
@@ -13,15 +13,12 @@ import "validate/validate.proto";
 
 // [#protodoc-title: TCP Proxy]
 // TCP Proxy :ref:`configuration overview <config_network_filters_tcp_proxy>`.
-// [#v2-api-diff: The route match now takes place in the :ref:`FilterChainMatch
-// <envoy_api_msg_FilterChainMatch>` table].
-
```

Comment:
You actually need this blank line, otherwise you lose the doc title and the link doesn't show at the top like the rest of the filter. (See generated docs)